tests/html5skin-test.js

```diff
@@ -352,6 +352,7 @@ describe('Controller', function() {
       controller.onSetEmbedCode('newEmbedCode');
       controller.onEmbedCodeChanged('newEmbedCode');
       expect(controller.skin.state.currentPlayhead).toBe(0);
+      expect(controller.state.multiAudio).toBe(null);
```

Comment:
Can we move this to a separate test in the multi-audio section?  It looks like `multiAudio` state is null by default, so this wouldn't check that the state is being cleared. The test would need to set a multi audio state before calling `onEmbedCodeChanged`. The changes look good otherwise, thanks for the update.

Reply:
\+